index.js

```diff
@@ -19,8 +19,19 @@ async function action() {
     if (!releases.length) {
       throw new Error(`No releases found in kong/deck`);
     }
-
-    version = releases[0].tag_name.replace(/^v/, "");
+    
+    for(let i=0; i < releases.length; i++) {
+      if(releases[i].prerelease) {
+        continue;
+      }
+      
+      version = releases[0].tag_name.replace(/^v/, "");
```

Comment:
```suggestion
      version = releases[i].tag_name.replace(/^v/, "");
```

Should this be `releases[i]`?

Reply:
ah yes, sorry typing too fast 😮‍💨